### PR TITLE
TW-761 Add liquidity baking

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

**Review by lendihop — DISMISSED** [review]

**Review by Boltman92 — DISMISSED** [review]

**Review by Boltman92 — APPROVED** [review]

File: src/modals/manage-farming-pool-modal/withdraw-form/use-withdraw-formik.ts [hunk 1]
```diff
@@ -41,26 +44,37 @@ const validationSchema: SchemaOf<WithdrawFormValues> = objectSchema().shape({
...
+  );
   const initialValues = useMemo(
     () => ({
-      amountOptionIndex: isDefined(farm) && farm.item.type !== PoolType.STABLESWAP ? 0 : 3,
+      amountOptionIndex:
+        isDefined(farm) &&
+        farm.item.type !== FarmPoolTypeEnum.STABLESWAP &&
+        depositAmountAtomic.gte(MINIMAL_DIVISIBLE_ATOMIC_AMOUNT)
+          ? 0
+          : 3,
```
Comment (herkoss) [inline src/modals/manage-farming-pool-modal/withdraw-form/use-withdraw-formik.ts:68]: Fix magic numbers here and in all places, please

**Review by Boltman92 — APPROVED** [review]

**Review by herkoss — APPROVED** [review]
GJ!

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_857

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_857

**Pull request merged**